megatron/core/transformer/moe/experts.py

```diff
@@ -1098,6 +1059,7 @@ def __init__(
         self.num_local_experts = num_local_experts
         self.local_experts = torch.nn.ModuleList()
         self.ep_group = pg_collection.ep
+        self.tp_group = pg_collection.expt_tp
```

Comment:
Q: Any reason to not use expt_tp_group to be consistent? Are there any code depend on the naming?